packages/experimental/mosaic/src/components/Card/CardActions.tsx

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { ReactNode } from 'react';
+
+import { mx } from '@dxos/react-components';
+
+import { Icon } from './util';
+
+export type CardActionsSlots = {
+  root?: {
+    className?: string;
+  };
+};
+
+export type CardActionsProps = {
+  slots?: CardActionsSlots;
+  children?: ReactNode;
+  gutter?: boolean;
+};
+
+export const CardActions = ({ slots = {}, gutter, children }: CardActionsProps) => {
+  return (
+    <div className={mx('flex h-[40px] px-2', slots.root?.className)}>
```

Comment:
The other slot props should be spread before `className`, `{...slots.root}`

Reply:
Ah, I see only `className` is permitted… is there a reason `ComponentPropsWithoutRef<'div'>` won’t do?